pkg/bundle/remote_test.go

```diff
@@ -89,30 +89,4 @@ func TestWriteAndRead(t *testing.T) {
 	if string(remoteContents) != "some-contents" {
 		t.Errorf("Expected image contents to be \"some-contents\" but found %s", string(remoteContents))
 	}
-
-	// Now fetch this image again and ensure it is read from the cache (there will be a second cache layer).
-	cachedImg, err := Read(actualRef, &cacheOptions)
-	if err != nil {
-		t.Fatal(err)
-	}
-
-	layers, err = cachedImg.Layers()
-	if err != nil {
-		t.Fatal(err)
-	}
-
-	// There should now be two layers because one is the cache layer.
-	if len(layers) != 2 {
-		t.Errorf("Found unexpected number of layers in cached image: %d", len(layers))
-	}
-
-	// Check the cache dir and ensure there is something in there.
-	infos, err := ioutil.ReadDir(tempDir)
-	if err != nil {
-		t.Fatal(err)
-	}
-
-	if len(infos) < 1 {
-		t.Error("expected at least 1 cache file but found none")
-	}
```

Comment:
Removed this as I don't understand what it tests.. and it fails.. 😛

Reply:
Keep the replace for go container registry here and it will work https://github.com/tektoncd/cli/pull/1448/files#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6L48

Reply:
Right but it doesn't make sense to me. What does *this* tests ? Why do we care that there is 1 or 2 cache file ? Are we just testing the upstream library (go-containerregistry) ?

Reply:
I *especially* don't want to keep that `replace`

Reply:
Yes this make sense, lets remove it